avocado/plugins/jobs.py

```diff
@@ -0,0 +1,232 @@
+            spawner = PodmanSpawner()
+
+        if spawner is None:
+            msg = ("Could not find the spawner for job {}. This command is "
+                   "experimental and only supported when job executed with "
+                   "the Spawner architecture.")
+            print(msg.format(job_id))
+            return exit_codes.AVOCADO_GENERIC_CRASH
+
+        for test in results_data.get('tests'):
+            test_id = test.get('id')
+            print("Downloading files for test {} ...".format(test_id))
+            try:
+                for filename, stream in spawner.stream_output(job_id, test_id):
+                    destination = os.path.join(destination, filename)
+                    self._save_stream_to_file(stream, destination)
+            except SpawnerException as ex:
+                print("Error: Failed to download: {}. Exiting...".format(ex))
+                return exit_codes.AVOCADO_GENERIC_CRASH
+
+    def handle_show_command(self, config, jobs_results):
+        """Called when 'avocado jobs show' command is executed."""
+
+        job_id = config.get('jobs.show.job_id')
+        results_file = jobs_results.get(job_id)
+        config_file = os.path.join(os.path.dirname(results_file),
+                                   'jobdata/args.json')
+        try:
+            results_data = self._get_data_from_file(results_file)
+        except FileNotFoundError as ex:
+            # Results data are important and should exit if not found
+            print(ex)
+            return exit_codes.AVOCADO_GENERIC_CRASH
+
+        try:
+            config_data = self._get_data_from_file(config_file)
+        except FileNotFoundError:
+            pass
+
+        data = {'Job id': job_id,
+                'Debug log': results_data.get('debuglog'),
+                'Spawner': config_data.get('nrun.spawner', 'unknown'),
+                '#total tests': results_data.get('total'),
+                '#pass tests': results_data.get('pass'),
+                '#skip tests': results_data.get('skip'),
+                '#errors tests': results_data.get('errors'),
+                '#cancel tests': results_data.get('cancel')}
+
+        # We could improve this soon with more data and colors
+        self._print_job_details(data)
+        self._print_job_tests(results_data.get('tests'))
+
+    def run(self, config):
+        results = {}
+
+        jobs_dir = get_logs_dir()
+        for result in glob(os.path.join(jobs_dir, '*/results.json')):
+            with open(result, 'r') as fp:
+                job = json.load(fp)
+                results[job['job_id']] = result
+
+        subcommand = config.get('jobs_subcommand')
+        if subcommand == 'list':
+            self.handle_list_command(results)
+        elif subcommand == 'show':
+            self.handle_show_command(config, results)
+        elif subcommand == 'get-output-files':
+            self.handle_output_files_command(config, results)
+        return exit_codes.AVOCADO_ALL_OK
```

Comment:
Your methods are returning `AVOCADO_*` exit code, but you are just ignoring and returning success here.